pkg/sfu/buffer/buffer.go

```diff
@@ -349,6 +349,13 @@ func (b *Buffer) Write(pkt []byte) (n int, err error) {
 		}
 	}
 
+	// libwebrtc will use 0 ssrc for probing, don't push the packet to pending queue to avoid memory increasing since
+	// the Bind will not be called to consume the pending packets. More details in https://github.com/pion/webrtc/pull/2816
+	if rtpPacket.SSRC == 0 {
+		b.Unlock()
+		return
+	}
```

Comment:
Does TWCC module handle 0 properly? That module uses the interceptor code. So, I am guessing that okay?

Reply:
Wondering if we should rtp stats module update to ensure that sequence number handling is okay.

Reply:
The twcc should be ok to handle it since pion set codec for 0 ssrc stream to first negotiated video codec.
It should has a standalone sequence range for 0 ssrc stream so we don't need to do stats here.

We can confirm these after the chrome change has been released.